scene/gui/tab_bar.cpp

```diff
@@ -1560,7 +1564,7 @@ void TabBar::ensure_tab_visible(int p_idx) {
 	if (!is_inside_tree() || !buttons_visible) {
 		return;
 	}
-	if (p_idx == -1 && _can_deselect()) {
+	if (p_idx == -1) {
```

Comment:
Couldn't those two returns be joined into one?

Reply:
That's true, will do